src/skin/legacyskinparser.cpp

```diff
@@ -388,6 +388,15 @@ QWidget* LegacySkinParser::parseSkin(const QString& skinPath, QWidget* pParent)
     m_pParent = pParent;
     QList<QWidget*> widgets = parseNode(skinDocument);
 
+    const QString hotcueSkinDefaultColor = m_pContext->variable("HotcueSkinDefaultColor");
+    if (!hotcueSkinDefaultColor.isNull()) {
+        QColor color;
```

Comment:
Use the named color ctor to create an immutable value:
const QColor color(hotcueSkinDefaultColor);